src/test/shell/bazel/apple/bazel_objc_test.sh

```diff
@@ -117,4 +117,60 @@ EOF
       fail "Timestamp of contents of archive file should be zero"
 }
 
+function test_strip_symbols() {
+  setup_objc_test_support
+
+  rm -rf ios
+  mkdir -p ios
+
+  cat > ios/unused.h <<EOF
+#import <Foundation/Foundation.h>
+@interface UnusedClass: NSObject
+@property (nonatomic, assign) bool someProperty;
+@end
+EOF
+
+  cat > ios/unused.m <<EOF
+#import "unused.h"
+@implementation UnusedClass
```

Comment:
Is `UnusedClass` needed for anything?

Should this test define an unused function and then call `nm` on the binary to confirm it has been stripped?

Reply:
Unused (like the name mentions) was meant to be the code that was stripped at the end. The idea was to test that the symbols are stripped, I'll add the `nm` call on the binary to verify

Reply:
Unused classes aren't stripped. The objc runtime metadata has a global list of all classes, and so there's always at least that one reference, which keeps them from being stripped.

Reply:
Got it, my knowledge of exactly what is stripped was a bit flawed but thanks for clarifying!

Reply:
Note: I'm holding off on merging this PR until you've figured out how to (better) show that the stripping happened or determine that it cannot easily be shown. Just let me know. :)

Reply:
@aragos - This test reflects that now 👍